app/components/apply-page/role-details.tsx

```diff
@@ -0,0 +1,110 @@
+const RoleDetailsSection = () => {
+  return (
+    <div
+      className="pl-8 pr-8"
+      style={{ paddingLeft: "120px", paddingRight: "120px" }}
+    >
+      <div style={{ display: "flex", alignItems: "center" }}>
+        <a
```

Comment:
Let's wrap this in a <Link>, https://remix.run/docs/en/main/components/link. You can use the relative prop to be able to go back one segment.